example/pubspec.yaml

```diff
@@ -5,8 +5,8 @@ publish_to: "none"
 version: 1.0.0+1
 
 environment:
-  sdk: ^3.1.0
-  flutter: ^3.13.0
+  sdk: ">=3.5.0 <4.0.0"
+  flutter: ">=3.24.3"
 
```

Comment:
in general probably we should not be so strict about this, but since this package is used only by clara I used the same config. Wdyt?

Reply:
> this package is used only by clara
afaik :)

Reply:
I aimed this comment for the main pubspec file though...